_sections/91-encrypted-forms.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Encryption 
+---
+Encrypted forms provide a mechanism to keep your data private even when using http: for communications (e.g., when you do not have an SSL certificate or https: is not available).
```

Comment:
In addition, it provides security for the duration in which the data is stored on a device and on the server.

Reply:
I would want to say "finalized data" actually, because that's a rather important thing.